website/addons/wiki/tests/test_wiki.py

```diff
@@ -569,6 +569,20 @@ def test_project_wiki_delete_w_valid_special_characters(self, mock_sharejs):
         self.project.reload()
         assert_not_in(to_mongo_key(SPECIAL_CHARACTERS_ALLOWED), self.project.wiki_pages_current)
 
+    @mock.patch('website.addons.wiki.utils.broadcast_to_sharejs')
+    def test_project_wiki_versions_after_delete(self, mock_sharejs):
+        # Ensure wiki versions do not reappear when a wiki is deleted and a new
```

Comment:
Try to use more descriptive test titles instead of comments. https://cosdev.readthedocs.io/en/latest/process/testing.html#general-testing-guidelines